clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3203,7 +3204,14 @@ private int processKafkaDataMessage(
             ChunkedValueManifest chunkedValueManifest = manifestSerializer.deserialize(
                 ByteUtils.extractByteArray(put.getPutValue()), // must be done before recordTransformer changes putValue
                 AvroProtocolDefinition.CHUNKED_VALUE_MANIFEST.getCurrentProtocolVersion());
-            hostLevelIngestionStats.recordAssembledValueSize(chunkedValueManifest.getSize(), currentTimeMs);
+            ByteBuffer rmdPayload = put.getReplicationMetadataPayload();
+            boolean isValueManifest = rmdPayload != null && rmdPayload.equals(VeniceWriter.EMPTY_BYTE_BUFFER);
+            if (isValueManifest) {
+              recordAssembledRecordSizeRatio(keyLen + chunkedValueManifest.getSize(), currentTimeMs);
+              hostLevelIngestionStats.recordAssembledValueSize(chunkedValueManifest.getSize(), currentTimeMs);
```

Comment:
The above statement is measuring both key and value and this line is only measuring value.
This is confusing, and I think we need to make them consistent, and I think it is fine to only measure value size for now by skipping `keyLen` as `key` is typically small and the chunking is only for value.

Reply:
What is confusing about these two? They are separate metrics that both utilize value manifests.
 
`recordAssembledValueSize()` is for an existing metric measuring assembled values that was added prior to this Handling Large Records project as part of a GCN action item.

`recordAssembledRecordSizeRatio()` is for comparing the current record's size with the `maxRecordSizeBytes` setting.  I thought this project was specifically about handling large records, which consist of key + value (and possible RMD)?

I actually had confusion even over the term "record" at the beginning, because there is also the concept of a record in Kafka.

Reply:
If it is fine to only measure value size, I will change the name to `recordAssembledValueSizeRatio()`.